test/libsolidity/semanticTests/abiEncoderV2/abi_encode_v2_in_modifier_used_in_v1_contract.sol

```diff
@@ -38,5 +38,5 @@ contract C is B {
 // compileViaYul: also
 // ----
 // test() -> 5, 10
-// gas irOptimized: 101019
+// gas irOptimized: 92642
```

Comment:
Saves 8k gas? Very surprising.

Reply:
Maybe the function call got inlined? Can you check?

Reply:
It's code size - the modifier creates a contract and this change saves quite a bit of code.